src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5901,6 +5901,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_UndecoratedCancellationTokenParameter_Title" xml:space="preserve">
     <value>Async-iterator member has one or more parameters of type 'CancellationToken' but none of them is decorated with the 'EnumeratorCancellation' attribute, so the cancellation token parameter from the generated 'IAsyncEnumerable&lt;>.GetAsyncEnumerator' will be unconsumed</value>
   </data>
+  <data name="ERR_MultipleEnumeratorCancellationAttributes" xml:space="preserve">
+    <value>The EnumeratorCancellationAttribute cannot be used on multiple parameters</value>
```

Comment:
>The EnumeratorCancellationAttribute cannot be used on multiple parameters [](start = 11, length = 73)

This is awkward english.  You appear to be using a type as a noun, but are referring to the uses of the type as an attribute.  Perhaps reword:

`The attribute [EnumeratorCancellation] cannot be used on multiple parameters`